deployments/charts/kuma/templates/cp-deployment.yaml

```diff
@@ -35,6 +35,7 @@ metadata:
   name: {{ include "kuma.name" . }}-control-plane
   namespace: {{ .Release.Namespace }}
   labels: {{ include "kuma.cpLabels" . | nindent 4 }}
+  annotations: {{ include "kuma.cpDeploymentAnnotations" . | nindent 4 }}
```

Comment:
I wonder if we should do the same thing with annotations as we do with `cpLabels`, so both `Pods` and the `Deployment` get the annotations?

Reply:
Good question. I thought about that, but in my use case (stakater reloader), I need to add annotations on the deployment only. But I agree that other use cases may need to put same annotations on resources.
My plan was to :
- use kuma.cpDeploymentAnnotations for deployment only annotations
- later use kuma.cpAnnotations for the same mechanism as kuma.cpLabels

Reply:
yeah. I think we should just have two separate annotations, one for Deployment, one for Pods.